types/splitpanes/index.d.ts

```diff
@@ -1,85 +1,198 @@
-// Type definitions for splitPanes 1.0
+// Type definitions for splitPanes 2.2
 // Project: https://github.com/antoniandre/splitpanes
-// Definitions by: teenhe <https://github.com/noonhorse>
+// Definitions by: n0k0 <https://github.com/n0k0>
```

Comment:
Did @noonhorse ask to be removed as an owner?

Reply:
Hi!
No, but this is a complete rewrite of a definition that seemed abandoned.

Reply:
That's fair 👍